collections/riscv-brazil/index.md

```diff
@@ -0,0 +1,11 @@
+---
+items:
+ - darklife/darkriscv
+ - zxmarcos/riscado-v
+ - racerxdl/riskow
+ - duinos/riscuinho
+display_name: Risc-V Cores Made Brazil
```

Comment:
```suggestion
display_name: Risc-V Cores Made in Brazil
```
Would this read better?

Reply:
Thanks.

Reply:
I have not yet used this Github change feature, can I assume that the way I proceeded here the suggested change was approved? or do I need to manually change my branch? or on the next merge will it be seen?

Reply:
Looks like you resolved this without accepting the suggestion. If you unresolve and commit the suggestion a commit will be made to your branch right in the GitHub UI.

Reply:
Para  mim há um botão que alterna de estado entre "resolved conversation"/"unresolved conversation", eu marquei como "resolved conversation" pois vejo como uma sugestão correta e aceita.

De qualquer fora irei no meu branch agora e farei o ajuste manualmente caso não atualise atuomáticamente.

Reply:
For me there is a button that alternates between "resolved conversation" / "unresolved conversation", I marked it as "resolved conversation" because I see it as a correct and accepted suggestion.

Anyway, I'll go to my branch now and do the adjustment manually if I don't update it automatically.